docs/production.md

```diff
@@ -11,3 +12,180 @@ configurations are available:
 | ----------------------- | ------------------------------- | -----------
 | OpenCensus Agent        | `OCAGENT`                       | Use OpenCensus.
 | Stackdriver\*           | `STACKDRIVER`                   | Use Stackdriver.
+
+
+## Rotating secrets
+
+This section describes how to rotate secrets in the system.
+
+### Cookie keys
+
+**Recommended frequency:** 30 days, on breach
+
+The cookie keys are an array. The items at odd indicies are HMAC keys and the
+items at even indicies are encryption keys. The HMAC key should be 64 bytes and
+the encryption key should be 32. Both keys are supplied to this system in
+base64, for example:
+
+```sh
+export COOKIE_KEYS="ARLaFwAqBGIkm5pLjAveJuahtCnX2NLoAUz2kCZKrScUaUkEaxHSvJLVYb5yAPCc441Cho5n5yp8jdEmy6hyig==,RLjcRZeqc07s6dh3OK4CM1POjHDZHC+usNU1w/XNTjM="
```

Comment:
Could. you add a command to turn this from base64? Just to make this as dumb simple as possible to follow in case of a breach.

Reply:
Not sure I understand. The `openssl` commands output base64, and the envvar accepts base64. Why would we want something that decodes it?

Reply:
Clarified.